resident/resident-service/src/test/resources/bootstrap.properties

```diff
@@ -1,154 +1,155 @@
-mosip.id.validation.identity.addressLine3.[*].value=^(?=.{0,50}$).*
-mosip.id.validation.identity.region.[*].value=^(?=.{0,50}$).*
-mosip.id.validation.identity.province.[*].value=^(?=.{0,50}$).*
-mosip.id.validation.identity.city.[*].value=^(?=.{0,50}$).*
-mosip.id.validation.identity.referenceIdentityNumber=^([0-9]{10,30})$
-# NEW Properties --------------------
-IDSchema.Version=0.1
-resident.identityjson=registration-processor-identity.json
-id.repo.update=mosip.id.update
-mosip.registration.processor.grace.period=10800
-id.repo.vidType=Perpetual
-token.request.appid=regproc
-token.request.clientId=mosip-regproc-client
-token.request.secretKey=abc123
-
-# imp
-objectstore.adapter.name=PosixAdapter
-mosip.kernel.registrationcenterid.length=5
-mosip.kernel.machineid.length=5
-mosip.commons.packetnames=id,evidence,optional
-# reader
-provider.packetwriter.resident=source:REGISTRATION_CLIENT,process:NEW|UPDATE|LOST|CORRECTION|ACTIVATED|DEACTIVATED|RES_UPDATE|LOST|RES_REPRINT,classname:io.mosip.commons.packet.impl.PacketReaderImpl
-# writer
-provider.packetwriter.mosip=source:REGISTRATION_CLIENT,process:NEW|UPDATE|LOST|CORRECTION|ACTIVATED|DEACTIVATED|RES_UPDATE|LOST|RES_REPRINT,classname:io.mosip.commons.packet.impl.PacketWriterImpl
-# can be OnlinePacketCryptoServiceImpl OR OfflinePacketCryptoServiceImpl
-objectstore.crypto.name=OfflinePacketCryptoServiceImpl
-default.provider.version=v1.0
-auth.server.validate.url=http://localhost
-mosip.kernel.xsdstorage-uri=http://localhost/
-mosip.kernel.xsdfile=mosip.xsd
-mosip.kernel.pdf_owner_password=1234
-
-#Auth Services
-CREDENTIAL_STATUS_URL=https://dev.mosip.net/v1/credentialrequest/get/
-CREDENTIAL_REQ_URL=https://dev.mosip.net/v1/credentialrequest/requestgenerator
-CREDENTIAL_CANCELREQ_URL=https://dev.mosip.net/v1/credentialrequest/cancel/
-CREDENTIAL_TYPES_URL=https://dev.mosip.net/v1/credentialservice/types
-
-PARTNER_API_URL=https://dev.mosip.net/v1/partnermanager/partners
+spring.cloud.config.uri=localhost
+spring.cloud.config.label=master
+spring.cloud.config.name=master
+spring.application.name=commons-packet-service
+spring.profiles.active=test
+management.endpoints.web.exposure.include=refresh
+APPLICATION_Id=DATASHARE
+PARTNER_REFERENCE_Id=mpartner-default-resident
+DECRYPT_API_URL=https://dev.mosip.net/v1/keymanager/decrypt
+KERNELENCRYPTIONSERVICE=https://dev.mosip.net/idauthentication/v1/internal/getCertificate
+crypto.PrependThumbprint.enable=true
+#management.security.enabled=false
+#--------------ID and Version --------------
+auth.internal.id=mosip.identity.auth.internal
+auth.internal.version=1.0
+mosip.registration.processor.print.id=mosip.registration.print
+mosip.registration.processor.application.version=1.0
+vid.create.id=mosip.vid.create
+resident.vid.version=v1
+resident.vid.id=mosip.resident.vid
+vid.revoke.id=vid.revoke.id
+resident.revokevid.id=mosip.resident.vidstatus
+auth.type.status.id=mosip.identity.authtype.status.update
+mosip.primary-language=eng
```

Comment:
here too. mosip.primary-language and mosip.secondary-language configs are being removed. Should not be referred anywhere in code.

Reply:
done